src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Dates.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using System.Security.Cryptography.X509Certificates
+@using Microsoft.AspNetCore.Mvc.ModelBinding
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.ProviderCommitments.Web.RouteValues
+@model SFA.DAS.ProviderCommitments.Web.Models.Apprentice.DatesViewModel
+
+@{
+    ViewData["Title"] = "New training start date";
+    ViewBag.GaData.Vpv = "/apprentices/change-employer/dates";
+}
+
+<partial name="_ValidationSummary" />
+
+<h1 class="govuk-heading-xl">New training start date</h1>
+
+<p class="govuk-body">The restarted date must be on or after the stop date.</p>
+
+<table class="govuk-table">
+    <tbody class="govuk-table__body">
+
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header"></th>
+            <td class="govuk-table__cell"></td>
+        </tr>
+
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header das-table-cell-width-50">Stop Date:</th>
```

Comment:
In the screenshot, the D is not capitalised and there is no colon. Also, the date itself should be formatted with full month name.

Reply:
corrected

Reply:
I asked Darryl while in Dev about the full month name, he said I can keep it like this as that's how we appear to have it displayed on the details page but can change it if required